packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -53,120 +68,56 @@ async function processTemplate(month, force) {
-      params: ['amount', null, 'month']
-    },
-    {
-      type: 'by',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) repeat every (\d+) months$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'repeat']
-    },
-    {
-      type: 'week',
-        re: /^#template \$?(\d+(\.\d{2})?) repeat every week starting (\d{4}\-\d{2}\-\d{2})$/im,//eslint-disable-line
-      params: ['amount', null, 'starting']
-    },
-    {
-      type: 'week',
-        re: /^#template \$?(\d+(\.\d{2})?) repeat every week starting (\d{4}\-\d{2}\-\d{2}) up to \$?(\d+(\.\d{2})?)$/im,//eslint-disable-line
-      params: ['amount', null, 'starting', 'limit']
-    },
-    {
-      type: 'weeks',
-        re: /^#template \$?(\d+(\.\d{2})?) repeat every (\d+) weeks starting (\d{4}\-\d{2}\-\d{2})$/im,//eslint-disable-line
-      params: ['amount', null, 'weeks', 'starting']
-    },
-    {
-      type: 'weeks',
-        re: /^#template \$?(\d+(\.\d{2})?) repeat every (\d+) weeks starting (\d{4}\-\d{2}\-\d{2}) up to \$?(\d+(\.\d{2})?)$/im,//eslint-disable-line
-      params: ['amount', null, 'weeks', 'starting', 'limit']
-    },
-    {
-      type: 'by_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) repeat every year$/im,//eslint-disable-line
-      params: ['amount', null, 'month']
-    },
-    {
-      type: 'by_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) repeat every (\d+) years$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'repeat']
-    },
-    {
-      type: 'spend',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2})$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from']
-    },
-    {
-      type: 'spend',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2}) repeat every (\d+) months$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from', 'repeat']
-    },
-    {
-      type: 'spend_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2}) repeat every year$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from']
-    },
-    {
-      type: 'spend_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2}) repeat every (\d+) years$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from', 'repeat']
-    },
-    {
-      type: 'percentage',
-      re: /^#template (\d+(\.\d+)?)% of (.*)$/im,
-      params: ['percent', null, 'category']
-    },
-    { type: 'error', re: /^#template .*$/im, params: [] }
-  ];
+const TEMPLATE_PREFIX = '#template ';
 
+async function getCategoryTemplates() {
   let templates = {};
 
-  let notes = await db.all(`SELECT * FROM notes WHERE note like '%#template%'`);
+  let notes = await db.all(
+    `SELECT * FROM notes WHERE note like '%${TEMPLATE_PREFIX}%'`
```

Comment:
We could add `WHERE lower(note)` to accomodate case insenstivity here.  I haven't thought of a reason this would break anything yet.

Reply:
```suggestion
    `SELECT * FROM notes WHERE lower(note) like '%${TEMPLATE_PREFIX}%'`
```